### PR TITLE
[ko] Update tags for Korean Docs

### PR DESCRIPTION
**Pull request by seokho-son**
### Describe your changes
- Update Tags for all Korean documents according to the English documents in Main branch.
- Update file name (microservices.md -> microservices-architecture.md)
- Tag: Fundamental ("핵심 개념" -> "기본 개념") considering the terms in the Fundamental catagory.

### Related issue number or link (ex: `resolves #issue-number`)
None

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
PTAL: @jihoon-seo @yunkon-kim 

And please use the Squash merge method. ;)

File: content/ko/service.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 title: 서비스 (Service)
 status: Completed
 category: 개념
-tags: ["아키텍처", "", ""]
+tags: ["아키텍처", "기본 개념", ""]
```
Comment (yunkon-kim) [inline content/ko/service.md:5]:
```suggestion
tags: ["애플리케이션", "기본 개념", ""]
```

`/en/service.md`의 태그(application, fundamental)을 수정 반영 하겠습니다 ^^
Ref) https://github.com/cncf/glossary/blob/main/content/en/service.md

**Review by yunkon-kim — APPROVED** [review]
LGTM :-)

**Comment by seokho-son** [comment]
PTAL @jihoon-seo :)

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Pull request merged**